### ISSUE TITLE
XML2JS Exception

### ISSUE THREAD

**Issue by TVilaboa** [Storj/storjshare-gui#593]
### Versions

Replace the values below with your own:

- **GUI Version:** V 5.0.1
- **Operating System:** Windows 10
### Expected Behavior

Please describe the program's expected behavior.

Space should be shared.

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

Sharing stops after a few seconds, even after multiple restarts. I have already installed, uninstalled and delete all files, at least 10 times. Tested with x32 and x64 versions. UPnP activated, ports forwarded and firewall open. It started getting peers only the first time i ran it.

`{"level":"warn","message":"you are not publicly reachable, trying traversal strategies...","timestamp":"2017-05-17T02:46:19.905Z"}
C:\Program Files (x86)\Storj Share\resources\app\node_modules\xml2js\lib\xml2js.js:216
          throw ex;
          ^

Error: Text data outside of root node.
Line: 78
Column: 1
Char: `

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Start Storj

**Comment by littleskunk** [comment]
```
root@storj:~# npm list -g xml2js
/usr/local/lib
└─┬ storjshare-daemon@2.5.4
  └─┬ storj-lib@6.4.2
    └─┬ nat-upnp@1.1.0
      └── xml2js@0.1.14
```

Your router gives a bad UPnP response. The xml2js is outdated. The issue was fixed in 0.4.2.

Please use manual port forwarding. I will open a pull request but I think this issue has low priority and we wait for the next nat-upnp release. For a low priority issue I don't like to implement a workaround and track nat-upnp releases in order to disable the workaround later.

**Comment by littleskunk** [comment]
Moved to https://github.com/indutny/node-nat-upnp/issues/31
